package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "@packt/rustici-sdk",
+  "version": "0.0.1",
+  "description": "Wrapper for Rustici API",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "files": [
+    "dist/**/*"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "lint": "tslint -p tsconfig.json",
+    "lint:fix": "tslint -p tsconfig.json --fix",
+    "test": "nyc mocha --require ts-node/register --require source-map-support/register --recursive 'tests/**/*.spec.ts'",
+    "clean": "rm -rf dist && rm -rf coverage && rm -rf .nyc_output",
+    "prepublishOnly": "npm run lint && npm run test && npm run build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@bitbucket.org/packt-internal/rustici-sdk.git"
+  },
+  "author": {
+    "name": "Packt Publishing Ltd",
+    "url": "https://www.packtpub.com/"
+  },
+  "contributors": [
+    {
+      "name": "Ashutosh Bane",
+      "email": "baneashutosh10@gmail.com"
+    }
+  ],
+  "engines": {
+    "node": ">=8.10"
+  },
+  "license": "Apache-2.0",
```

Comment:
I thought it was MIT?

Reply:
I am not sure as well.
@ukmadlz Can you please help us here.

Reply:
Just change line 36 to "license": "MIT",